skyportal/handlers/api/source.py

```diff
@@ -1130,6 +1142,65 @@ class Validator(Schema):
                     isouter=True,
                 )
 
+        if localization_dateobs is not None:
```

Comment:
same question about logic when this is None...

Reply:
Same deal here, just the original query.